src/cpuinfo_x86.c

```diff
@@ -1097,9 +1097,11 @@ static void ParseCpuId(const uint32_t max_cpuid_leaf, X86Info* info) {
     features->ssse3 = IsBitSet(leaf_1.ecx, 9);
     features->sse4_1 = IsBitSet(leaf_1.ecx, 19);
     features->sse4_2 = IsBitSet(leaf_1.ecx, 20);
+    features->sse4a = IsBitSet(leaf_7.ecx, 6);
```

Comment:
IIRC, leaf_7 is EAX=7
while here we want EAX=80000001h ie leaf_80000001 ?

Reply:
I really apologize for that. I made a mistake due to inattention.